### ISSUE TITLE
"It Came from Planet Zilog" is super busted

### ISSUE THREAD

**Issue by waddlesplash** [mgba-emu/mgba#249]
pouet: http://www.pouet.net/prod.php?which=65345
YouTube: https://www.youtube.com/watch?v=If2xW3IW-LI

Screen on mGBA is almost always a solid color and some weird artifacts now and then until the scene with the flying saucer (although that scene has visual glitches). After that, it seems OK for a time, until after the "PHANTASY" logo exits, and then it's busted again until the twisting things. Continues to go in and out for the rest of the demo.

Colors appear off vs. the video though, not sure what's up with that. Sound seems correct though.

All I got in the log was
```
[STUB] GB I/O:	Writing to unknown register FF15:00
[STUB] GB I/O:	Writing to unknown register FF1F:00
```
repeated a bunch, and then later
```
[STUB] GB MBC:	MBC5 unknown value 15
[STUB] GB MBC:	MBC5 unknown value 14
[STUB] GB MBC:	MBC5 unknown value 13
[STUB] GB MBC:	MBC5 unknown value 12
[STUB] GB MBC:	MBC5 unknown value 11
[STUB] GB MBC:	MBC5 unknown value 10
[STUB] GB MBC:	MBC5 unknown value 0F
[STUB] GB MBC:	MBC5 unknown value 0E
[STUB] GB MBC:	MBC5 unknown value 0D
[STUB] GB MBC:	MBC5 unknown value 0C
[STUB] GB MBC:	MBC5 unknown value 0B
[STUB] GB MBC:	MBC5 unknown value 09
[STUB] GB MBC:	MBC5 unknown value 08
[STUB] GB MBC:	MBC5 unknown value 07
[STUB] GB MBC:	MBC5 unknown value 06
[STUB] GB MBC:	MBC5 unknown value 05
```
and more of the like.

**Comment by endrift** [comment]
Mostly working as of 4159f8ea6305f32d34dc47e8001eb166a255a04e.

**Comment by endrift** [comment]
Five issues left in this:

- [ ] GB loading screen is missing
- [ ] Every other reset, the first starfield is laggy
- [ ]The first starfield seems a bit glitchy regardless
- [ ] The rotating 3D city is missing
- [ ] Alien silhouette screen is broken

**Comment by endrift** [comment]
Only one issue remaining, where some scanlines get corrupted in two of the scenes.

**Comment by endrift** [comment]
Scanline latching fixed in 0083fad9663157e2745e363896644267c5155105, fixing the last major bug with this.
